components/Text/index.ts

```diff
@@ -4,4 +4,4 @@ import { default as Data } from './Data';
 import { default as Heading } from './Heading';
 import { default as PositionType } from './PositionType';
 
-export default { Body, Data, Heading, BigText, PositionType };
+export { Body, Data, Heading, BigText, PositionType };
```

Comment:
What do you think of going with a few foundational text components of:
```
Heading
Body
Number
```
and potentially `Label`

And then all other sub styles becomes variants on those components?

Reply:
I think that `Number` can just be something like `<Body mono />` as well. I'm reluctant to remove BigText for now, even though it's technically equivalent to `Heading` with the added color and logo, because I don't want us to have the logo svg thing within the Heading component. Outside of that, I think the current state is pretty much equivalent to this. PositionType should eventually be taken out of the `Text` folder, but I still can't find the right place for it.

Reply:
Ok maybe we can rename `BigText` to `LogoText` or something and change any non logo use of BigText to a Heading? PositionType if it's specific to Futures we could move to under futures components?

Reply:
Good idea. I just pushed some code that does the first part. But since `PositionType` isn't specific to futures (we use it on the dashboard), I don't think we can move it under the futures components.

Reply:
If it's used in a futures related section of the dashboard I think it makes sense to add under futures still. If it's completely unrelated then maybe components/shared folder?